### PR TITLE
Improvement #497: add timeout option to helm_cli Ansible custom module

### PR DESCRIPTION
**Pull request by alexandre-allard**

### CODE REVIEW

**Review by ballot-scality — COMMENTED** [review]
You can commit with keywords : https://help.github.com/articles/closing-issues-using-keywords/
To fixes the issue

**Comment by alexandre-allard** [comment]
@ballot-scality Oh nice, thanks for the advice, I'll try this.

File: roles/helm_common/library/helm_cli.py [hunk 1]
```diff
@@ -77,6 +78,9 @@ def install(self):
         namespace = self.params.get('namespace')
         if namespace:
             cmd.extend(['--namespace', namespace])
+        timeout = self.params.get('timeout')
```
Comment (Zempashi) [inline roles/helm_common/library/helm_cli.py:81]: Could we add also --timeout to command to supports it (helm delete) ? Could be done as separated PR if needed

Comment (alexandre-allard) [reply roles/helm_common/library/helm_cli.py:81]: It makes sense, I'll add it.

**Review by Zempashi — APPROVED** [review]

**Comment by alexandre-allard** [comment]
@bert-e approve

**Pull request merged**